### PR TITLE
Use the TAG being built in db-controller chart values

### PR DESCRIPTION
**Pull request by cxvisa**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -272,6 +273,7 @@ build-properties-crd:
 	@sed 's/{CHART_FILE}/$(CHART_FILE_CRD)/g' build.properties.in > crd.build.properties
 
 build-chart:
+	sed -i "/  tag: \"latest\"/c\  tag: ${TAG}" ${DB_CONTROLLER_CHART_VALUES}
```
Comment (daniel-garcia) [inline Makefile:276]: doing this inline will lead to someone checking in the actual tag then this won't work anymore. I suggest using a .in file or following appVersion see
https://stackoverflow.com/questions/58065225/image-tag-in-values-yaml-vs-appversion-in-chart-yaml

**Review by daniel-garcia — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (cxvisa) [reply Makefile:276]: remove the values.yaml file from git and have values.in file that generates the values.yaml file?  Did I understand it right?

Comment (cxvisa) [reply Makefile]: Added values.in file and removed values.yaml file from the git repo.  Now the values.yaml file gets generated on the fly.  We now set AppVersion in the Chart during package time (it is set to the TAG computed).  The deployment image version is picked from .Values.image.tag and if it is not set default value is set to.Chart.AppVersion.

**Review by daniel-garcia — APPROVED** [review]
LGTM

**Pull request merged**